projects/1.16.1/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2083 @@
+    "item.tetra.modular.broken": "损坏的",
+    "item.tetra.modular.empty_slot": "空",
+    "item.tetra.modular.magic_capacity.label": "魔力容量",
+    "item.tetra.modular.magic_capacity.description": "组件可以容纳与已容纳的魔力量。\n \n§7魔力容量：§e%s\n§7已使用：§e%s",
+    "item.tetra.modular.magic_capacity.description_extended": "魔力容量可以被超出，但加工这个组件时会消耗经验。改动物品时附魔可能会变得不稳定并导致负面效果。",
+    "item.tetra.modular.repair_material.label": "维修材料：",
+    "item.tetra.modular.repair_material.empty": "不需要维修",
+    "block.tetra.basic_workbench": "工作台",
+    "block.tetra.basic_workbench.description": "用锤子右击合成台以创建工作台",
+    "block.tetra.rack": "架子",
+    "block.tetra.rack.description": "物品可以被挂在架子上。用右键放入或取出物品。\n在工作台附近挂在架子上的工具就像在玩家背包里一样可以用于合成，架子需要在 3x3x3 的范围内才能起效（以工作台向上一个方块为中心）。",
+    "block.tetra.block_geode": "晶洞方块",
+    "item.tetra.geode": "晶洞玉石",
+    "item.tetra.geode.tooltip": "在工作台上破开，可能得到有价值的材料\n \n§8掉落物受时运附魔影响",
+    "tetra.break_geode_action.label": "破开",
+    "item.tetra.pristine_diamond": "无瑕钻石",
+    "tetra.shatter_diamond_action.label": "打碎",
+    "item.tetra.pristine_emerald": "无瑕绿宝石",
+    "tetra.shatter_emerald_action.label": "打碎",
+    "item.tetra.pristine_lapis": "无瑕青金石",
+    "tetra.shatter_lapis_action.label": "打碎",
+    "item.tetra.pristine_gem.description": "在用于改进物品时产生更多增益。可以在工作台上破碎，可能会产出普通的品质的宝石\n \n§8掉落物受时运附魔影响",
+    "item.tetra.dragon_sinew": "龙筋",
+    "item.tetra.magmatic_cell": "热力单元",
+    "item.tetra.magmatic_cell.charge": "充能：%s",
+    "item.tetra.magmatic_cell.charge_full": "§a满",
+    "item.tetra.magmatic_cell.charge_good": "§a好",
+    "item.tetra.magmatic_cell.charge_low": "§e低",
+    "item.tetra.magmatic_cell.charge_empty": "§c空",
+    "block.tetra.forged_pillar": "锻造支柱",
+    "block.tetra.forged_platform": "锻造平台",
+    "block.tetra.forged_platform_slab": "锻造平台",
+    "block.tetra.forged_wall": "锻造墙",
+    "block.tetra.forged_vent": "锻造通风口",
+    "block.tetra.forged_crate": "锻造板条箱",
+    "block.tetra.forged_container": "锻造容器",
+    "tetra.forged_container.compartment": "隔间 %s",
+    "block.tetra.hammer_base": "锻造锤",
+    "block.tetra.hammer_head.name": "锤头",
+    "block.tetra.forged_workbench": "锻造工作台",
+    "block.tetra.transfer_unit": "传输单元",
+    "block.tetra.core_extractor": "地核提取器",
+    "block.tetra.extractor_piston": "提取活塞",
+    "block.tetra.extractor_pipe": "提取管道",
+    "block.tetra.seeping_bedrock": "渗流基岩",
+    "block.multiblock_hint.1x2x1": "§8[§71x2x1 多方块§8]",
+    "item.tetra.forged_bolt": "螺栓",
+    "tetra.salvage_bolt_action.label": "回收",
+    "item.tetra.forged_beam": "结构横梁",
+    "tetra.salvage_beam_action.label": "回收",
+    "item.tetra.forged_mesh": "柔性网格",
+    "tetra.salvage_mesh_action.label": "回收",
+    "item.tetra.quick_latch": "快卸销",
+    "tetra.salvage_quick_latch_action_hammer.label": "回收",
+    "tetra.salvage_quick_latch_action_pry.label": "回收",
+    "item.tetra.metal_scrap": "金属废料",
+    "item.tetra.metal_scrap.description": "奇形怪状的金属，它们的并无用处或用途未知，等待你赋予它们全新的价值",
```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
    "item.tetra.metal_scrap.description": "奇形怪状的金属，它们用途未知或已无用处，并在等待你赋予它们全新的价值",
```